src/code/portal.js

```diff
@@ -166,6 +164,18 @@ export default class WasabeePortal {
     return n;
   }
 
+  get faked() {
+    return this.id.length != 35 || this.id == this.name;
+  }
+
+  get loading() {
+    return this.id.length == 35 && this.id == this.name;
+  }
+
+  get pureFaked() {
+    return this.id.length != 35;
```

Comment:
we should probably set a constant in static.js instead of hardcoding "35" -- use that same constant in the auxiliar.js for the ID generation (and a few other places -- loading ops from localStorage comes to mind)